amr-wind/utilities/sampling/DTUSpinnerSampler.H

```diff
@@ -51,7 +51,8 @@ public:
 
 #ifdef AMR_WIND_USE_OPENFAST
 
-    void bcast_turbine(double* /*turbine_pack*/, int /*root_proc*/);
+    void bcast_turbine(
+        amrex::Array<double, 18>& /*turbine_pack*/, int /*root_proc*/);
```

Comment:
This changed from a double to an array. 
Is this the desired behavior?
What is the 18?

Reply:
it took a pointer to a double array. Now I just pass the actual array around (well really just a reference to it) and that way we get all the bounds checking, etc that is in `Array`.